packages/core/src/factory.ts

```diff
@@ -62,12 +64,30 @@ export function factory<Resource, Input, Output, Config extends IPrismConfig>(
     config: Config,
     validations: IPrismDiagnostic[]
   ): TE.TaskEither<Error, ResourceAndValidation & { output: Output }> => {
-    const mockCall = () =>
-      components.mock({
+    const mockCall = () => {
```

Comment:
in the event `mockOrForward`'s call sites include the mock config, this should be all reverted